README.md

```diff
@@ -40,29 +41,28 @@ NOTE: this code assumes you are using a reference DEM (and other airborne lidar
 
 -b buffer_meters Total width for the transform area [Default: 3]
 
--g geoid         Is the reference DEM CRS orthometric (geoid height)? Will be auto set to True if you don't supply a DEM [Default: False]
-
 ```
 
+NOTE: this code assumes you are using a reference DEM (and other airborne lidar data) referenced to the same vertical datum (ellipsoid vs. geoid). If your vertical datum differs between the two you can use the `geoid_tool.py` to correct this.
 
-#### Optional: If estimating grain size, install R and the following packages 
+## Transforming vertical datum
 
-See `ice-road-pipeline.py` for the additional optional flags. This currently requires a target with a known reflectance in order to calibrate lidar reflectance. It assumes wavelength = 1064 nm.
+As stated above, if your vertical datum differs between your point cloud and reference DEM use the following to correct your reference,
 
 ```
-library(raster)
-library(lidR)
-library(rlas)
-library(dplyr)
-library(readr)
-library(terra)
-library(sf)
-library(data.table)
-
+$ python scripts/geoid_tool.py <path-to-reference-data> -e epsg -t transform_command -a <dir-to-ASP-bin> -d debug
 ```
 
+### Flags
 
-###  Additional information :books:
+```
+-e epsg                   Desired EPSG code you want (for example, 32611)
+-t transform_command      Options: to_geoid , to_ellipsoid
+-a asp_dir                Directory with ASP binary files
```

Comment:
Is this optional or required?
I assume it would automatically pick up ASP once it is inside the IRC folder?

Reply:
ASP does not necessarily need to be installed in IRC the way we currently have it . which is why i believe it is required.

Reply:
Correct me on my perspective and there is no co-registration without ASP, so we need to require it to make ICP function. We also make it an explicit point in the README

Reply:
@jomey I think we are confusing eachother with how I am saying it... **It is required**. I am just saying the ASP bin is currently allowed to be saved anywhere on your machine.

And if we were to make the change you are saying (where its ../ASP/bin/) , it would require changing the pipeline code too. 

As for BSU implementation of it, I am cautious to make this change here in this PR because I do not know where they have ASP installed on Borah/Kelvin.

Reply:
Ahhh ... got it.
Yeah, lets not require it. Maybe in this case simplify the instructions on how to install ASP. Mainly thinking of removing the step to rename the folder (step 2.) and leave it up to user where it goes.

Currently on Borah you can load as a module (not quite sure how up to date it is) and also just have it installed in your local home directory.